tests/phpunit/tests/functions/wpAdminNotice.php

```diff
@@ -1,326 +1,326 @@
-			),
-			'data attribute with unsafe value'          => array(
-				'message'  => 'A notice with an additional attribute with an unsafe value.',
-				'args'     => array(
-					'attributes' => array( 'data-unsafe' => '<script>alert( "Howdy, admin!" );</script>' ),
-				),
-				'expected' => '<div class="notice" data-unsafe="&lt;script&gt;alert( &quot;Howdy, admin!&quot; );&lt;/script&gt;"><p>A notice with an additional attribute with an unsafe value.</p></div>',
-			),
-			'additional invalid attribute'              => array(
-				'message'  => 'A notice with an additional attribute that is invalid.',
-				'args'     => array(
-					'attributes' => array( 'not-valid' => 'not-valid' ),
-				),
-				'expected' => '<div class="notice"><p>A notice with an additional attribute that is invalid.</p></div>',
-			),
-			'multiple attributes with "role", invalid, data-*, numeric, and boolean' => array(
-				'message'  => 'A notice with multiple attributes with "role", invalid, "data-*", numeric, and boolean.',
-				'args'     => array(
-					'attributes' => array(
-						'role'      => 'alert',
-						'disabled'  => 'disabled',
-						'data-name' => 'my-name',
-						'data-id'   => 1,
-						'hidden',
-					),
-				),
-				'expected' => '<div class="notice" role="alert" data-name="my-name" data-id="1" hidden><p>A notice with multiple attributes with "role", invalid, "data-*", numeric, and boolean.</p></div>',
-			),
-			'paragraph wrapping as a falsy value rather than (bool) false' => array(
-				'message'  => 'A notice with paragraph wrapping as a falsy value rather than (bool) false.',
-				'args'     => array(
-					'paragraph_wrap' => 0,
-				),
-				'expected' => '<div class="notice"><p>A notice with paragraph wrapping as a falsy value rather than (bool) false.</p></div>',
-			),
-		);
-	}
-
-	/**
-	 * Tests that `_doing_it_wrong()` is thrown when a 'type' containing spaces is passed.
-	 *
-	 * @ticket 57791
-	 *
-	 * @expectedIncorrectUsage wp_get_admin_notice
-	 */
-	public function test_should_throw_doing_it_wrong_with_a_type_containing_spaces() {
-		ob_start();
-		wp_admin_notice(
-			'A type containing spaces.',
-			array( 'type' => 'first second third fourth' )
-		);
-		$actual = ob_get_clean();
-
-		$this->assertSame(
-			'<div class="notice notice-first second third fourth"><p>A type containing spaces.</p></div>',
-			$actual
-		);
-	}
-
-	/**
-	 * Tests that `wp_admin_notice()` fires the 'wp_admin_notice' action.
-	 *
-	 * @ticket 57791
-	 */
-	public function test_should_fire_wp_admin_notice_action() {
-		$action = new MockAction();
-		add_action( 'wp_admin_notice', array( $action, 'action' ) );
-
-		ob_start();
-		wp_admin_notice( 'A notice.', array( 'type' => 'success' ) );
-		ob_end_clean();
-
-		$this->assertSame( 1, $action->get_call_count() );
-	}
-}
+<?php
+
+/**
+ * Tests for `wp_admin_notice()`.
+ *
+ * @group admin
+ *
+ * @covers ::wp_admin_notice
+ */
+class Tests_Admin_WpAdminNotice extends WP_UnitTestCase {
```

Comment:
```suggestion
 * @group functions.php
 *
 * @covers ::wp_admin_notice
 */
class Tests_Functions_WpAdminNotice extends WP_UnitTestCase {
```
Couple of minor changes to match the new location. 🙂